src/main/java/com/inventorysystem/inventory/dao/ProductDAOStub.java

```diff
@@ -23,28 +23,28 @@
     @Override
     public Product addItem(Product product) throws Exception {
         Integer productID = product.getProductID();
-        allProudcts.put(productID, product);
+        allProducts.put(productID, product);
         return product;
     }
 
     /**
      * Displays all Products in the list.
-     * @return
+     * @return a List containing values from allProducts HashMap
```

Comment:
Added detail to comments where doable to reduce lack of comments, one of the seven SonarQube Sins. [Reference](https://docs.sonarqube.org/display/SONARQUBE45/Not+Enough+or+Too+Many+Comments

Reply:
Very nice... the @param and @return get ignored too often in JavaDoc!